packages/2018/src/components/NewHomePage.js

```diff
@@ -0,0 +1,212 @@
+            technology, use your skills to give back, and work with a dream team
+            of talented people, you’ll love working on CIVIC. Collaborating on a
+            project is full of creative learning opportunities and experimental
+            R&D.
+            <br />
+            <br />
+            We’re looking for developers, designers, content creators, product
+            managers, community organizers and more to build with us. It all
+            starts by filling out an application, and we’ll be in touch when we
+            have a match for a project that fits your skills and interests.
+          </p>
+          <div className="section-button-container">
+            <Button
+              display="inline"
+              margin="0 10px 0 0"
+              bkgndColor={BrandColors.subdued.rgba}
+            >
+              APPLY NOW
+            </Button>
+            <Button
+              display="inline"
+              margin="0 0 0 0"
+              bkgndColor={BrandColors.subdued.rgba}
+            >
+              LEARN MORE
+            </Button>
+          </div>
+          <img
+            className="placeholder-section-image"
+            src={placeholderContributorsImg}
+            alt="placeholder"
+          />
+        </div>
+
+        <h2>ABOUT</h2>
+
+        <div className="home-section" css={sectionPlatformLayout}>
+          <h3>CIVIC Platform</h3>
+          <p>
+            The CIVIC Platform is designed for making public information public
+            knowledge. In order to do that, we build applications to provide a
+            set of common frameworks for rapid development.
+            <br />
+            <br />
+            CIVIC is open-source which means it’s non-proprietary, and you’re
+            free to replicate or contribute to our code to develop with your own
+            teams or resources. If you’ve a developer, you don’t have to join a
+            team to start contributing.
+          </p>
+          <h4 className="platform-h4">CIVIC IS COMPRISED OF:</h4>
+          <div className="link-container">
+            <a className="platform-link" href="http://service.civicpdx.org/">
+              EXPLORE DATA
+            </a>
+            <a
+              className="platform-link"
+              href="https://github.com/hackoregon?type=source"
+            >
+              EXPLORE CODE
+            </a>
+            <a
+              className="platform-link"
+              href="https://hackoregon.github.io/civic/"
+            >
+              DOCUMENTATION
+            </a>
+          </div>
+          <img
+            css={css`
+              margin-left: -15px;
+              width: 100%;
+            `}
+            src={placeholderPlatformImg}
+            alt="placeholder"
+          />
+        </div>
+
+        <div className="home-section" css={sectionCivicLayout}>
+          <h3>Civic Software Foundation</h3>
```

Comment:
At large browser font sizes, this will wrap and cause overlap. We should find a way to avoid this